zkevm-circuits/src/evm_circuit/execution/error_code_store.rs

```diff
@@ -82,13 +82,7 @@ impl<F: Field> ExecutionGadget<F> for ErrorCodeStoreGadget<F> {
             vec![1.expr(), 2.expr()],
         );
 
-        let common_error_gadget = CommonErrorGadget::construct_with_return_data(
-            cb,
-            opcode.expr(),
-            5.expr(),
-            memory_address.offset(),
-            memory_address.length(),
-        );
+        let common_error_gadget = CommonErrorGadget::construct(cb, opcode.expr(), 5.expr());
```

Comment:
We could see [the implementation of geth](https://github.com/ethereum/go-ethereum/blob/master/core/vm/instructions.go#L614-L618), it only assigns `returnData` when this call reverts. Otherwise it clear the `returnData`. So, here we shouldn't assign `offset` and `length`.